sleap_io/model/skeleton.py

```diff
@@ -21,7 +28,15 @@ class Node:
         name: Descriptive label for the landmark.
     """
 
-    name: str
+    name: str = field()
```

Comment:
Do not need the `field` - `@define` will  do the following: 

> If any attributes are annotated and no unannotated [attrs.fields](https://www.attrs.org/en/stable/api.html#attrs.fields)s are found, it assumes auto_attribs=True.

where

> auto_attribs ([bool](https://docs.python.org/3/library/functions.html#bool)) –
 If True, collect [PEP 526](https://www.python.org/dev/peps/pep-0526/)-annotated attributes (Python 3.6 and later only) from the class body.
 In this case, you must annotate every field. If attrs encounters a field that is set to an [attr.ib](https://www.attrs.org/en/stable/api.html#attr.ib) but lacks a type annotation, an [attr.exceptions.UnannotatedAttributeError](https://www.attrs.org/en/stable/api.html#attrs.exceptions.UnannotatedAttributeError) is raised.

However, we still need `attr.ib` in cases where we call decorators on the method:
> For features that use the attribute name to create decorators (e.g. [validators](https://www.attrs.org/en/stable/init.html#validators)), you still must assign [attr.ib](https://www.attrs.org/en/stable/api.html#attr.ib) to them. Otherwise Python will either not find the name or try to use the default value to call e.g. validator on it.